EnglishFilter/sections/specific.txt

```diff
@@ -308,6 +308,7 @@ herdeaths.net##.e3lan-post > a[href]
 iafd.com##.bancol
 rlsbb.to###text-5
 rlsbb.to###text-11
+vegasslotsonline.com##.game-box__panel
```

Comment:
It is related to the rule above, please rewrite it via '#$#'.

vegasslotsonline.com#$#.game-box__panel { display: none !important; }